s3-vfs/src/main/java/org/pentaho/s3common/S3CommonFileObject.java

```diff
@@ -224,10 +224,21 @@ public void doAttach() throws Exception {
           injectType( FileType.IMAGINARY );
           return;
         }
+
+        // One common case is "403 Access Denied", for bucket names which exist in the same region
+        // but are not of this account. This can also happen for normal files and is being handled
+        // similarly in handleAttachExceptionFallback.
+        // Any other errors should also bubble up.
+
+        // Make sure this gets printed for the user.
+        logger.error( "Could not get information on " + getQualifiedName(), e );
+        throw new FileSystemException( "vfs.provider/get-type.error", getQualifiedName(), e );
```

Comment:
@nawaz34-hitachivantara can you explain the code change to removing this thrown exception?